string transformed_frame_id
geometry_msgs/PoseArray poses
---
geometry_msgs/PoseArray poses_transformed